packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -107,10 +113,30 @@ export const FeedContainer = ({
     insaneMode: listMode,
     loadedSettings,
   } = useContext(SettingsContext);
+  const { user, isLoggedIn } = useAuthContext();
   const { shouldUseFeedLayoutV1 } = useFeedLayout();
   const { feedName } = useActiveFeedNameContext();
   const router = useRouter();
   const onboardingV4dot5 = useFeature(feature.onboardingV4dot5);
+  const userAcquisitionVersion = useFeature(feature.userAcquisition);
```

Comment:
Using useFeature here, it would mean we are enrolling all users landing in all feed pages (since all feed pages use feed container) to the experiment. We want to avoid that, we want to enroll only users landing on feed container only after onboarding(so on my feed page). 
In order to do this, we can check for the feature on onboarding page, where we set ua=true.

Reply:
OK, so if I understand it correctly, we want to add `useFeature` hook to the onboarding page as well. That way, we make sure to enroll first, and the `useFeature` here is just going to check the enrolled value. Is that a correct assumption?

Reply:
Yup, so using useFeature just in onboarding page